charts/warpstream-agent/templates/deployment.yaml

```diff
@@ -94,11 +98,8 @@ spec:
             {{- end }}
             {{- if and .Values.resources (not .Values.goMaxProcs) }}
             {{- if .Values.resources.requests }}
-            {{- if not (regexMatch "^[0-9]+$" (toString .Values.resources.requests.cpu)) }}
-              {{- fail "Only use whole numbers for cpu request because we are using it to set GOMAXPROCS." }}
-            {{- end }}
             - name: GOMAXPROCS
-              value: {{ .Values.resources.requests.cpu | quote }}
+              value: {{ $goMaxProcs | quote }}
```

Comment:
did you confirm this value is getting set correctly? I always get confused about "block scoping" of variables in helm

Reply:
yeah I tested with different values and it looks good to me